### PR TITLE
Unifies some method names with other threshold projects

### PR DESCRIPTION
**Pull request by vzotova**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [X] Other

**Required reviews:** 
- [ ] 1
- [X] 2
- [ ] 3

**What this does:**
Unifies naming, see threshold-network/solidity-contracts#157
Breaks  ABI

### CODE REVIEW

File: contracts/contracts/TACoApplication.sol [hunk 1]
```diff
@@ -611,7 +618,7 @@ contract TACoApplication is IApplication, ITACoChildToRoot, OwnableUpgradeable {
     /**
      * @notice Returns staking provider for specified operator
      */
-    function stakingProviderFromOperator(address _operator) external view returns (address) {
+    function stakingProviderToOperator(address _operator) external view returns (address) {
```
Comment (cygnusv) [inline contracts/contracts/TACoApplication.sol:621]: This should be operatorToStakingProvider, isn't?

Comment (vzotova) [reply contracts/contracts/TACoApplication.sol:621]: of course, my bad

Comment (vzotova) [reply contracts/contracts/TACoApplication.sol]: ✔️

**Review by derekpierre — COMMENTED** [review]
Looks good - but this will have downstream effects on `nucypher` agents (TACoApplicationAgent/TACoChildApplicationAgent). This PR needs a matching PR in `nucypher` with the relevant updates before merging.

**Review by theref — APPROVED** [review]
This one makes me happy! :D 🚀

**Review by derekpierre — APPROVED** [review]
🎸

File: contracts/contracts/TACoApplication.sol [hunk 2]
```diff
@@ -618,16 +625,14 @@ contract TACoApplication is IApplication, ITACoChildToRoot, OwnableUpgradeable {
     /**
      * @notice Returns staking provider for specified operator
      */
-    function stakingProviderFromOperator(address _operator) external view returns (address) {
+    function operatorToStakingProvider(address _operator) external view returns (address) {
```
Comment (cygnusv) [inline contracts/contracts/TACoApplication.sol:628]: This will require changes to nucypher. Opened https://github.com/nucypher/nucypher/issues/3350

**Review by cygnusv — APPROVED** [review]
Looks good, but will need a very small change in nucypher (not urgent)

**Pull request merged**